api.md

```diff
@@ -278,7 +278,8 @@ as a table with column headings.
 
 >:WARNING: The use of the `migrate_data` argument to convert a non-empty table can
 lock the table for a significant amount of time, depending on how much data is
-in the table.
+in the table. It can also run into deadlock if foreign key constraints exist to other
+tables.
```

Comment:
Suggestion, reverse the order to make it easier to read:

> Depending on the amount of data in the table, use of `migrate_table` can cause long lock times. Using `migrate_data` can also cause deadlocks if the table have foreign key constraints to other tables.

Reply:
I don't want to modify the text, which was before me. It will just increase amount of discussions.
IMHO the sentence must start with `migrate_data` to make it clear that it is related to usage of this argument.